netlist/scheme/gnet-gsch2pcb.scm.in

```diff
@@ -182,8 +184,15 @@
   (let* ((c2p (pipe))  ; child to parent
          (p2c (pipe))) ; parent to child
 
+    ; TODO: make setvbuf( port, mode ) work with both Guile 2.0 and 2.2
```

Comment:
Have you thought about using a conditional here? Say, you could check guile `effective-version` or `minor-version` to opt appropriate `mode` value. E.g.
```
(setvbuf (cdr c2p) (if (string= (minor-version) "0") _IONBF 'none))

Reply:
@vzh Thank you. :)